src/m365/spo/commands/file/file-sharinglink-list.ts

```diff
@@ -1,26 +1,24 @@
 import { Logger } from '../../../../cli/Logger';
 import GlobalOptions from '../../../../GlobalOptions';
-import request, { CliRequestOptions } from '../../../../request';
-import { formatting } from '../../../../utils/formatting';
 import { odata } from '../../../../utils/odata';
+import { spo } from '../../../../utils/spo';
 import { validation } from '../../../../utils/validation';
 import SpoCommand from '../../../base/SpoCommand';
 import commands from '../../commands';
-import { GraphFileDetails } from './GraphFileDetails';
 
 interface CommandArgs {
   options: Options;
 }
 
-interface Options extends GlobalOptions {
+export interface Options extends GlobalOptions {
   webUrl: string;
   fileId?: string;
   fileUrl?: string;
   scope?: string;
 }
 
 class SpoFileSharingLinkListCommand extends SpoCommand {
-  private static readonly scope: string[] = ['anonymous', 'users', 'organization'];
+  public readonly allowedScopes: string[] = ['anonymous', 'users', 'organization'];
```

Comment:
Is there a reason why we are changing this line?

Reply:
This was due to me adding it to the util file in the beginning. I have changed this back to how it was :-)